trino/dbapi.py

```diff
@@ -391,6 +395,16 @@ def _execute_prepared_statement(
         sql = 'EXECUTE ' + statement_name + ' USING ' + ','.join(map(self._format_prepared_param, params))
         return trino.client.TrinoQuery(self._request, query=sql, legacy_primitive_types=self._legacy_primitive_types)
 
+    def _execute_immediate_statement(self, statement: str, params):
+        """
+        Binds parameters and executes a statement in one call.
+
+        :param statement: sql to be executed.
+        :param params: parameters to be bound.
+        """
+        sql = 'EXECUTE IMMEDIATE ' + self._format_prepared_param(statement) + ' USING ' + ','.join(map(self._format_prepared_param, params))
```

Comment:
why do you call `_format_prepared_param` on the `statement`? Shouldn't it just be `statement.replace("'", "''")`?

Reply:
I just wanted to make sure I am using the same logic for formatting string literals as elsewhere.  I realize now it's just escaping single quotes, but I was looking at https://github.com/trinodb/trino/blob/4ab52e4a5e2abd31d7b28aa506dd788e11f62091/core/trino-parser/src/main/java/io/trino/sql/ExpressionFormatter.java#L1054 and was thinking that maybe at one point we'll augment the Python client to handle Unicode, as well.

Reply:
hmmm, I don't think we have tests for unicode (or byte-array) literals. But I think it's the server who "knows" that a `U&` is a unicode literal - client passes it as a plain string in other clients too as well.

Reply:
My impression was that Unicode characters needed to be encoded as `U&` in string literals, but I guess I was wrong.